### ISSUE TITLE
Tensorflow Exception when running in internal mode with tensorflow-gpu

### ISSUE THREAD

**Issue by Atom-101** [Unity-Technologies/ml-agents#347]
I get the following error in unity console when running the agent in internal mode. 

TFException: NodeDef mentions attr 'output_dtype' not in
Op<name=Multinomial; signature=logits:T, num_samples:int32 ->
output:int64; attr=seed:int,default=0; attr=seed2:int,default=0;
attr=T:type,allowed=[DT_FLOAT, DT_DOUBLE, DT_INT32, DT_INT64,
DT_UINT8, DT_INT16, DT_INT8, DT_UINT16, DT_HALF]; is_stateful=true>;
NodeDef: multinomial/Multinomial = Multinomial[T=DT_FLOAT,
output_dtype=DT_INT64, seed=0, seed2=0](dense_3/MatMul,
multinomial/Multinomial/num_samples). (Check whether your
GraphDef-interpreting binary is up to date with your
GraphDef-generating binary.).
TensorFlow.TFStatus.CheckMaybeRaise (TensorFlow.TFStatus
incomingStatus, System.Boolean last) (at
<6ed6db22f8874deba74ffe3e566039be>:0)
TensorFlow.TFGraph.Import (TensorFlow.TFBuffer graphDef,
TensorFlow.TFImportGraphDefOptions options, TensorFlow.TFStatus
status) (at <6ed6db22f8874deba74ffe3e566039be>:0)
TensorFlow.TFGraph.Import (System.Byte[] buffer,
TensorFlow.TFImportGraphDefOptions options, TensorFlow.TFStatus
status) (at <6ed6db22f8874deba74ffe3e566039be>:0)
TensorFlow.TFGraph.Import (System.Byte[] buffer, System.String prefix,
TensorFlow.TFStatus status) (at <6ed6db22f8874deba74ffe3e566039be>:0)
CoreBrainInternal.InitializeCoreBrain () (at
Assets/ML-Agents/Scripts/CoreBrainInternal.cs:112)
Brain.InitializeBrain () (at Assets/ML-Agents/Scripts/Brain.cs:187)
Academy.Awake () (at Assets/ML-Agents/Scripts/Academy.cs:124)

Training happens smoothly.

PS: This error started appearing once I switched to tensorflow-gpu. In CPU mode I didn't get this error.

**Comment by Atom-101** [comment]
The solution is to use tensorflow-gpu 1.4.0.
While upgrading to tensorflow-gpu my tensorflow version got updated to 1.5.1 from 1.4.0
Using this version one can train smoothly but when playing in internal mode there is probably a conflict with the Unity TensorflowSharp plugin leading to this error.
